class11.1/main.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <vector>
+#include <cstdlib>
+#include <ctime>
+
+void proposeBook(std::vector<std::string>& books) {
```

Comment:
using output argument is a bad practice - you remove elements from that vector which is not correspond to function name